UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Index.cshtml

```diff
@@ -69,6 +69,7 @@
 
 <script type="text/javascript">
     var serverData = @Html.Raw(Newtonsoft.Json.JsonConvert.SerializeObject(@Model.YearAndWeekList));
+    console.log(@Html.Raw(Newtonsoft.Json.JsonConvert.SerializeObject(@Model.YearAndWeekList)));
```

Comment:
Please remove this console.log statement. I believe it was used for local testing purpose.

Reply:
Done